adafruit_stmpe610.py

```diff
@@ -1,26 +1,54 @@
-# SPDX-FileCopyrightText: 2017 Jerry Needell for Adafruit Industries
+# SPDX-FileCopyrightText: 2017, 2022 Jerry Needell for Adafruit Industries
 #
 # SPDX-License-Identifier: MIT
-
 """
 `adafruit_stmpe610`
 ====================================================
-
 This is a CircuitPython Driver for the STMPE610 Resistive Touch sensor
 
-* Author(s): Jerry Needell
-"""
+* Author(s): Jerry Needell, CedarGroveMakerStudios
+
+Implementation Notes
+--------------------
+**Hardware:**
+
+**Software and Dependencies:**
 
-# imports
+* Adafruit CircuitPython firmware for the supported boards:
+  https://github.com/adafruit/circuitpython/releases
+"""
 
 import time
 from micropython import const
 
 
-__version__ = "0.0.0-auto.0"
+__version__ = "1.2.7"
```

Comment:
If I am not mistaken, tis should not be changed. The version is set when the library is released.

Reply:
I didn't change it in the submittal. Must be an artifact of the auto PR process.